### ISSUE TITLE
Caching with Realm and RxJava/RetroFit

### ISSUE THREAD

**Issue by seitensei** [TeamGFS/GFSClock#72]

**Comment by oneSIX** [comment]
this is going to be a bugger, and also needs flow.

**Comment by oneSIX** [comment]
this may be a subtask of the API current API.

**Comment by seitensei** [comment]
Yeah. @oneSIX does the current API have any way to run verification? My understand is that, at some points, the server will 'fail' and give us a bad reply- we can take that to mean that an interaction didn't succeed, but honestly, I'd prefer something more well designed and repeatable.

**Comment by oneSIX** [comment]
Would be nice is rather than a failure the server replied back at the least with a error code / message.

**Comment by seitensei** [comment]
We probably won't be using RxJava at all, since we don't need to merge datasets.

The medium post that @oneSIX found deals with data caching so that the app is usable offline completely. We're not maintaining that kind of cache and not bringing down data and merging it with our own copy.
